clickhouse_driver/columns/boolcolumn.py

```diff
@@ -0,0 +1,26 @@
+from .base import FormatColumn
+
+
+class BoolColumn(FormatColumn):
+    ch_type = 'Bool'
+    py_types = (bool, )
+    format = 'B'
```

Comment:
There is `?` format which will cast value to bool. Why we shouldn't use it? Explicit `bool` cast in `before_write_items` will be unnecessary.

Reply:
I missed its existence, and got hung up on the fact that it's `UInt8` under the hood in CH. Thank you; this also lets me remove `after_read_items`.